OracleTuxedo/core/dockerfiles/22.1.0.0.0/tuxedo22.1.0.0.0.rsp

```diff
@@ -0,0 +1,502 @@
+
+LDAP_SUPPORT_SSL=<Value Unspecified>
+
+#-------------------------------------------------------------------------------
+#Name       : INSTALL_SSL
+#Datatype   : Boolean
+#Description: This variable is to choose install SSL or not.
+#Component  : tuxedoClientCore
+#-------------------------------------------------------------------------------
+
+INSTALL_SSL=<Value Unspecified>
+
+#-------------------------------------------------------------------------------
+#Name       : LDAP_FILTER_FILE
+#Datatype   : String
+#Description: This variable is for configure LDAP filter file
+#Component  : tuxedoClientCore
+#-------------------------------------------------------------------------------
+
+LDAP_FILTER_FILE=<Value Unspecified>
+
+#-------------------------------------------------------------------------------
+#Name       : LDAP_CONFIG
+#Datatype   : StringList
+#Description: This variable is for configure LDAP parameters(Service Name, PortID, BaseObject)
+#Component  : tuxedoClientCore
+#-------------------------------------------------------------------------------
+
+LDAP_CONFIG=<Value Unspecified>
+
+#-------------------------------------------------------------------------------
+#Name       : TLISTEN_PORT
+#Datatype   : String
+#Description: This variable is for configure tlisten port
+#Component  : tuxedoServerBase
+#-------------------------------------------------------------------------------
+
+TLISTEN_PORT="3050"
+
+#-------------------------------------------------------------------------------
+#Name       : INSTALL_SAMPLES
+#Datatype   : Boolean
+#Description: This variable is for configure whether need to install Tuxedo samples
+#Component  : tuxedoServerBase
+#-------------------------------------------------------------------------------
+
+INSTALL_SAMPLES=<Value Unspecified>
+
+#-------------------------------------------------------------------------------
+#Name       : ENCRYPT_CHOICE
+#Datatype   : Number
+#Description: This variable is for configure encrypt method, 0 is LLE, 1 is SSL.
+#Component  : tuxedoServerBase
+#-------------------------------------------------------------------------------
+
+ENCRYPT_CHOICE=<Value Unspecified>
+
+#-------------------------------------------------------------------------------
+#Name       : CONFIG_TLISTEN
+#Datatype   : Boolean
+#Description: To configure the tlisten paramters at installation phase, set this variable to true.
+#Component  : tuxedoServerBase
+#-------------------------------------------------------------------------------
+
+CONFIG_TLISTEN=<Value Unspecified>
+
+#-------------------------------------------------------------------------------
+#Name       : TLISTEN_PASSWORD
+#Datatype   : String
+#Description: This variable is for configure tlisten password
+#Component  : tuxedoServerBase
+#-------------------------------------------------------------------------------
+
+TLISTEN_PASSWORD="oracle"
```

Comment:
Please remove this hard-coded password and replace it with a randomly generated one.